bucket/swctl.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.9.0",
+    "description": "SkyWalking CLI is a command interaction tool for the SkyWalking user or OPS team",
+    "homepage": "https://github.com/apache/skywalking-cli/",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.apache.org/skywalking/cli/0.9.0/skywalking-cli-0.9.0-bin.tgz",
+            "hash": "sha512:9a0445650c1fc55c8cfc53080742842abc5b36120dfc2f6adf430fc7211049637de362fdabf653929a52b0495327ad4ecd5c803c47abc19ada088ec73f26a19b"
+        }
+    },
+    "bin": "swctl.exe",
+    "pre_install": [
+        "Move-Item \"$dir\\skywalking-cli-*\\bin\\*-win*\" \"$dir\\swctl.exe\"",
+        "Remove-Item \"$dir\\skywalking-cli-*\" -Recurse"
```

Comment:
```suggestion
        "Rename-Item \"$dir\\bin\\swctl-*-win*\" \"swctl.exe\"",
        "Remove-Item \"$dir\\bin\\swctl-*\""
```

Reply:
The downloaded binaries are called `skywalking-cli-*`, but to keep it consistent with their build output and their documentation, I will rename it to `swctl.exe`